exercises/forth/README.md

```diff
@@ -52,3 +52,7 @@ In the test suites all tests but the first have been skipped.
 
 Once you get a test passing, you can enable the next one by
 changing `xtest` to `test`.
+
+
+## Submitting Incomplete Solutions
+It's possible to submit an incomplete solution so you can see how others have completed the exercise.
```

Comment:
Any reason for this to be in added specifically for this one exercise?

Reply:
This is coming from `config/exercise_readme.go.tmpl`.  You are not seeing it already in other READMEs?

Reply:
Ohh right, did not check other READMEs. The PR diff made me think it was added only for this exercise.